Documentation/network/kubernetes/requirements.rst

```diff
@@ -37,10 +37,8 @@ Enable CNI in Kubernetes
 ========================
 
 :term:`CNI` - Container Network Interface is the plugin layer used by Kubernetes to
-delegate networking configuration. CNI must be enabled in your Kubernetes
-cluster in order to install Cilium. This is done by passing
-``--network-plugin=cni`` to kubelet on all nodes. For more information, see
-the `Kubernetes CNI network-plugins documentation <https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/>`_.
+delegate networking configuration. Prior to Kubernetes 1.24, the CNI plugins could also be managed by the kubelet using the ``cni-bin-dir`` and ``network-plugin`` command-line parameters. These command-line parameters were removed in Kubernetes 1.24, with management of the CNI no longer in scope for kubelet.
+ For more information, see the `Kubernetes CNI network-plugins documentation <https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/>`_. To enable the cilium CNI plugin you need to configure the CNI plugin located at ``/etc/cni/net.d/`` . you can do this by making a config file for at the designated location . for making a config file you can refer to : `cilium config documentation <https://docs.cilium.io/en/stable/configuration/>`_. 
```

Comment:
At this point, the document is no longer really listing a requirement and instead providing generic background information. In that spirit, we should document that Ciliium requires a CRI-compliant runtime that uses CNI for networking. In that vein, Cilium is tested to work with cri-o and containerd.

Reply:
@squeed sorry for being inactive I was busy in some chores . Do you mean to say that I should remove all of it and just put "Cilium requires a CRI-compliant runtime that utilizes CNI for networking. It has been tested to work with cri-o and containerd. Ensure that your container runtime is compatible with Cilium."

Reply:
That would be my preference. This document's purpose is to capture Cilium's requirements, not identify Kubernetes upgrade concerns.